code/modules/jobs/job_types/agent.dm

```diff
@@ -142,6 +142,7 @@
 	head = /obj/item/clothing/head/hos/beret
 	ears = /obj/item/radio/headset/heads/agent_captain/alt
 	l_pocket = /obj/item/commandprojector
+	r_pocket = /obj/item/announcementmaker/lcorp
```

Comment:
They don't actually fit in pockets (I know it's lame)

Reply:
Guh